src/allmydata/immutable/checker.py

```diff
@@ -1,3 +1,15 @@
+"""
+Ported to Python 3.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from future.utils import PY2
+if PY2:
+    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
+
```

Comment:
I didn't notice this one, but made the explicit decision in a later PR that yeah, if it's just debug logs or whatever, switching to bytes or `repr()` instead of custom-massaged string formatting of byte sequences etc. is fine. A human can interpret what's going on reasonably.